week1/react-exercises/src/components/Hobbies.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react'
```

Comment:
`Component` in my IDE seems a quiet grey dark colour, which means it's been defined but never used, if no need for it, it's  better not confusing your self with many imports.
```suggestion
import React from 'react'
```